### PR TITLE
Max3

### PR DESCRIPTION
**Pull request by j-devy**

### CODE REVIEW

File: solutions/sum.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const sum = (a, b) => {
+  return a + b;
+};
+module.exports = sum;
```
Comment (MadhuBeesetty) [inline solutions/sum.js]: why do you have your previous branch files in your new branch?

File: solutions/max3.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const max3 = (num1, num2, num3) => {
```
Comment (MadhuBeesetty) [inline solutions/max3.js]: what if i have n number of numbers?

File: solutions/sum.js (hunk 1 above)
Comment (j-devy) [reply solutions/sum.js]: uhh not sure. what should i have instead?

File: solutions/max3.js (hunk 2 above)
Comment (j-devy) [reply solutions/max3.js]: max3 is suppose to take in 3 arguments, and return the biggest one, no?

**Pull request merged**